LondonBritishLibrary/orient/BLorient752.xml

```diff
@@ -1,15 +1,16 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient752" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Qalemǝnṭos, Didascalia Apostolorum, Faws Manfasāwi, The teaching of Apostles regarding the passion, Crucification and Resurrection of our Lord
-                    
-                    
+                <title>Qalemǝnṭos, Didǝsqǝlyā, Maṣḥafa faws manfasāwi,
+                  The teaching of Apostles regarding the passion, Crucification and Resurrection of our Lord
+
                 </title>
                 <editor key="SG"/>
```

Comment:
i would take out SG here, a change element for creating a stub is enough

Reply:
Well, he inserted also Information on the provenance, and on the addition